### PR TITLE
RK-7200 - New welcome page

### PR DESCRIPTION
**Pull request by LiranLast**
What do you think?

We'll also need some supporting screenshots and an architecture image.

### CODE REVIEW

File: docs/welcome.md [hunk 1]
```diff
@@ -4,13 +4,54 @@ title: Getting Started
...
+
+When working on a remote enviorment, it's imperative to know exactly what source code it's executing.
+
+Rookout integrates with your source-control provider to fetch you the correct source code for each and every enviorment ([security information)](https://www.rookout.com/security/source-code-security).
+
+## Integrate with your observability tools
+
+When investigating application issues, existing observability tools such as logging, tracing and error tracking, are an important source of information.
+
+Rookout integrates with those services to offer show you the correspending transaction identifiers.
```
Comment (jhendrick) [inline docs/welcome.md:35]: Rookout integrates with those services to show you the corresponding transaction identifiers.
(Remove offer and fix corresponding spelling)

File: docs/welcome.md [hunk 2]
```diff
@@ -4,13 +4,54 @@ title: Getting Started
...
+
+Rookout even empowers you to add logs, metrics, and alerts to running applications on the fly, seeing them side-by-side with traditional logs and metrics.
+
+## Key features
+
+1. Debug multiple application simultaneously.
```
Comment (jhendrick) [inline docs/welcome.md:41]: applications

File: docs/welcome.md [hunk 3]
```diff
@@ -4,13 +4,54 @@ title: Getting Started
...
+1. Negligible performance overhead with strict controls.
+1. Enterprise grade [security](https://www.rookout.com/security).
+
+## How does it work?
+
+Rookout's SDKs and agents use bytecode-manipulation (and similar techniques) to instantly add the instructions for data collection to the application's code as soon as you add a non-breaking breakpoint.
+
+## What's next?
 
-You can also try the following:
```
Comment (jhendrick) [inline docs/welcome.md:14]: Should we also put a link to the GitHub training page which has a lot of detailed explanations?  Up to you - I'm ok either way if we want to keep that separate as well just for trainings. - https://github.com/Rookout/rookout-training

**Comment by jhendrick** [comment]
Overall looks awesome, this will be a nice first entry into the docs vs. whats there today.

File: docs/welcome.md (hunk 3 above)
Comment (LiranLast) [reply docs/welcome.md:14]: Sure, why not?

**Comment by LiranLast** [comment]
> Overall looks awesome, this will be a nice first entry into the docs vs. whats there today.

@jhendrick Want to make the changes and merge?

**Review by jhendrick — APPROVED** [review]

**Pull request merged**